src/java/org/apache/cassandra/db/tries/TrieWalker.java

```diff
@@ -22,7 +22,7 @@
  * CompletableFuture.
  * See TrieDumper for sample usage.
  */
-interface TrieWalker<T, V>
+public interface TrieWalker<T, V>
```

Comment:
Note: @jacek-lewandowski will create a separate PR to merge this commit. We can review that PR and no need to review this commit. Jacek, please link your PR  here when you have it.